app/controllers/addresses_controller.rb

```diff
@@ -0,0 +1,21 @@
+class AddressesController < ApplicationController
+  def edit
+    @address = Address.find params[:id]
+  end
+
+  def update
+    @address = Address.find(params[:id])
+    if @address.update(address_params)
+      flash[:notice] = 'Erfogreich geupdated'
+      redirect_to player_path @address.addressable
```

Comment:
because address is polymorphic, use (more) Rails magic here and `redirect_to @address.addressable`

see: https://github.com/cfb-hasenheide/cfb-hasenheide/blob/master/app/controllers/comments_controller.rb#L25